features/admin.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.gis import admin
+from parler.admin import TranslatableAdmin
+
+from features.models import Feature
+
+
+@admin.register(Feature)
+class FeatureAdmin(TranslatableAdmin, admin.OSMGeoAdmin):
```

Comment:
This view is absolutely magical for such little code 👍 . If there's an easy way to extend it to support the "overridden" data model we could save a lot of development effort on the admin ui for the time being.